packages/@ama-sdk/core/src/plugins/gateway-token/readme.md

```diff
@@ -0,0 +1,9 @@
+> **Warning**: This pluging is deprecated and will be removed on v10.
```

Comment:
You could add a word about what to use instead (as we have in typescript file: please use AmadeusGatewayTokenRequestPlugin from @ama-sdk/amadeus-gateway-sdk instead).